packages/prism-react-renderer/src/components/useGetLineProps.ts

```diff
@@ -0,0 +1,20 @@
+import { ThemeDict } from "../utils/themeToDict"
+import { useCallback } from "react"
+import { LineInputProps, LineOutputProps } from "../types"
+import clsx from "clsx"
+
+export const useGetLineProps = (themeDictionary?: ThemeDict) =>
+  useCallback(
+    ({ className, style, line, ...rest }: LineInputProps) => {
+      const output: LineOutputProps = {
+        ...rest,
+        className: clsx("token-line", className),
+      }
+
+      if (themeDictionary != null) output.style = themeDictionary.plain
```

Comment:
This `!=` throws me off a bit, although I think it was in the previous code too. Would it be safe to just do `if (!themeDictionary)`?

Reply:
Yeah, I think `!=` is just checking for `undefined`/`null`, but `!` should work too.